elements/campaign-brand-messaging/campaign-brand-messaging.js

```diff
@@ -0,0 +1,46 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    if (!data.brand_messaging) {
+      console.log('Brand messaging not set on this campaign');
+      return;
+    }
+
+    this.innerHTML =
+      `<div class='campaign-brand-messaging'>
+        <span>${data.brand_messaging}</span>
+      </div>`;
+  }
+
+  handleRequestError (error) {
+    console.log(error);
+  }
+
+  attachedCallback () {
+    invariant(!!this.hasAttribute('src'), 'campaign-brand-messaging component requires a src');
```

Comment:
can this just be `this.hasAttribute('src')`? Doesn't it already return a boolean?